pyproject.toml

```diff
@@ -60,7 +60,8 @@ lint.ignore = [
   'PLR2004',
   'PTH',
   'RET505',
-  'S',
+  'S101',    # Won't fix
```

Comment:
I personally would aim to fix this within the library, and ignore the rule in `tests/` folder by adding

```
[tool.ruff.lint.per-file-ignores]
"tests/*" = ["S101"]
```